frontend/src/components/common/GenericModal.tsx

```diff
@@ -189,6 +190,10 @@ const StyledModal = styled(Modal)`
     .close {
       color: white;
       opacity: 1;
+      font-size: 3rem;
```

Comment:
corrections to the "x" icon on the modal.

Reply:
yeah - that icon always seemed a bit small. thanks for fixing